nativescript-angular/directives/list-view-comp.ts

```diff
@@ -242,7 +246,12 @@ export interface ComponentView {
 export type RootLocator = (nodes: Array<any>, nestLevel: number) => View;
 
 export function getItemViewRoot(viewRef: ComponentView, rootLocator: RootLocator = getSingleViewRecursive): View {
-    const rootView = rootLocator(viewRef.rootNodes, 0);
+    let rootView = rootLocator(viewRef.rootNodes, -1);
+    if (rootView instanceof ProxyViewContainer) {
```

Comment:
I think a more borad check should be used:
` if (!args.view instanceof LayoutBase || args.view instanceof ProxyViewContainer)`
This will handle cases when template produces a single `Label` or `Button` for example